packages/valory/skills/decision_maker_abci/behaviours/sampling.py

```diff
@@ -151,8 +151,8 @@ def async_act(self) -> Generator:
         """Do the action."""
         with self.context.benchmark_tool.measure(self.behaviour_id).local():
             idx = self._sample()
-            benchmarking_finished = False
-            day_increased = False
+            benchmarking_finished = None
+            day_increased = None
```

Comment:
This was causing an issue in the **normal flow**.

Reply:
Still trying to understand why None is a valid value and False not.

Reply:
Because otherwise the NONE event cannot ever be emitted from the round.